tests/integration_tests/test_applications_from_config.py

```diff
@@ -134,6 +135,42 @@ def compare_ecsv_files(file1, file2, tolerance=1.0e-5):
             assert np.allclose(table1[col_name], table2[col_name], rtol=tolerance)
 
 
+def assert_file_type(file_type, file_name):
+    """
+    Assert that the file is of the given type.
+
+    Parameters
+    ----------
+    file_type: str
+        File type (json, yaml).
+    file_name: str
+        File name.
+
```

Comment:
I would include the raise statements as well

Reply:
This function does not raise Exceptions, it catches them and adopts the return value accordingly. So I think no need to document it, as these exceptions will not be visible.